__tests__/ExpensiMark-HTML-test.js

```diff
@@ -1491,3 +1490,91 @@ test('Mention', () => {
     testString = '@user@DOMAIN.com';
     expect(parser.replace(testString)).toBe('<mention-user>@user@DOMAIN.com</mention-user>');
 });
+
+describe('edit mode', () => {
```

Comment:
Even though this is the terminology I've used to explain the task, I would refrain from using "edit mode" here since ExpensiMark is stateless and thus has no context of editing. Let's substitute it with "when should keep whitespace flag is enabled" or something related to it.